Add Frontend path prefix note to V19 upgrade guide (#279)
vaadin/docs: articles/guide/upgrading.asciidoc

The helpers also bypass framework internal and static resources (`/VAADIN/**`, `sw.js` ...).
Previously, these had to be explicitly matched and ignored in the app.

== Changes in Vaadin 19

These instructions apply when upgrading from a version before Vaadin 19.

=== TypeScript Configuration Now Includes Frontend Path Alias

The default `tsconfig.json` content was changed to introduce the `Frontend` import path alias.

> The content of the `tsconfig.json` file will not be updated automatically if it existed before the migration.

Reviewer: ```suggestion
The content of the `tsconfig.json` file is not updated automatically if it existed before the migration.
```

If you do not have any own modifications in this file, you can delete the old
`tsconfig.json` file.

> `vaadin-maven-plugin` will then automatically create the file with the new defaults next time when building the project or running the development mode.

Reviewer: ```suggestion
`vaadin-maven-plugin` creates the file automatically with the new defaults next time when building the project or running the development mode.
```
